controllers/storagecluster/cephcluster.go

```diff
@@ -1341,3 +1345,31 @@ func getOsdCount(sc *ocsv1.StorageCluster) int {
 	}
 	return osdCount
 }
+
+func determineDefaultCephDeviceClass(foundDeviceClasses []rookCephv1.DeviceClasses, isReplica1 bool, replica1DeviceClasses []string) string {
```

Comment:
`sc.Status.FailureDomainValues` only contains replica 1 device classes? Those don't seem like the same thing to me, but maybe it's just a naming issue.

Reply:
Yes for each failure domain value we have created one osd with the deviceclass as the failuredomainvalue.